docs/contributors.md

```diff
@@ -13,69 +13,95 @@ to see who has been involved already.
-  - Cesar Ruiz-Meraz, Raja Ravipati
-- **Oracle**
-  - Chad Arimura - [@carimura](https://github.com/banisadr)
-  - Stanley Halka - [@shalka](https://github.com/banisadr)
-  - Travis Reeder - [@treeder](https://github.com/banisadr)
-- **Red Hat**
-  - Jim Curtis - [@jimcurtis64](https://github.com/jimcurtis2)
-  - William Markito Oliveira - [@william_markito](https://github.com/markito)
-  - Gunnar Morling - [@gunnarmorling](https://github.com/gunnarmorling/)
-  - Tihomir Surdilovic - [@tsurdilo](https://github.com/tsurdilo)
-  - Lance Ball - [@lance](https://github.com/lance)
-- **SAP**
-  - Nathan Oyler - [@notque](https://github.com/notque)
-  - Stevo Slavić - [@sslavic](https://github.com/sslavic)
-  - Klaus Deissner - [@deissnerk](https://github.com/deissnerk)
-- **Serverless Inc**
-  - Austen Collins - [@ac360](https://github.com/ac360)
-  - Rupak Ganguly - [@rupakg](https://github.com/rupakg)
-  - Brian Neisler - [@brianneisler](https://github.com/brianneisler)
-  - Jeremy Coffield, Ganesh Radhakirshnan
-- **SolarWinds**
-  - Lee Calcote - [@leecalcote](https://github.com/leecalcote)
-- **VMWare**
-  - Mark Peek - [@markpeek](https://github.com/markpeek)
-- **Individuals**
-  - Rémi Cattiau - [@loopingz](https://github.com/loopingz)
-  - Jem Day - [@JemDay](https://github.com/JemDay)
-  - Vladimir Bacvanski
+# A
+- Sarah Allen - [@ultrasaurus](https://github.com/ultrasaurus)
+- Chad Arimura - [@carimura](https://github.com/banisadr)  
+- Neil Avery - [@bluemonk3y](https://github.com/bluemonk3y)
+
+# B
+- Vladimir Bacvanski
+- Lance Ball - [@lance](https://github.com/lance)
+- Bahram Banisadr - [@banisadr](https://github.com/banisadr)
+- Michael M Behrendt - [@mbehrendt](https://github.com/mbehrendt)
+- Thomas Bouldin - [@inlined](https://github.com/inlined)
+
+# C
+- Rémi Cattiau - [@loopingz](https://github.com/loopingz)
+- Lee Calcote - [@leecalcote](https://github.com/leecalcote)
+- Jeremy Coffield
+- Austen Collins - [@ac360](https://github.com/ac360)
+- Jim Curtis - [@jimcurtis64](https://github.com/jimcurtis2)
+
+# D
+- Doug Davis - [@duglin](https://github.com/duglin)
+- Jem Day - [@JemDay](https://github.com/JemDay)
+- Klaus Deissner - [@deissnerk](https://github.com/deissnerk)
+- Jie Ding - [@jieding](https://github.com/jieding)
+- Heng Du - [@duhenglucky](https://github.com/duhenglucky)
+
+# F
+- Louis Fourie - [@lfourie](https://github.com/lfourie)
+
+# G
+- Rupak Ganguly - [@rupakg](https://github.com/rupakg)
+- Arun Gupta
+
+# H
+- Stanley Halka - [@shalka](https://github.com/banisadr)
+- Morgan Hallmon
+- Yaron Haviv - [@yaronha](https://github.com/yaronha)
+- Robert-Jan Huijsman
+
+# K
+- Daniel Krook - [@krook](https://github.com/krook)
+
+# L
+- Rob Leidle
+- David Lyle - [@dklyle](https://github.com/dklyle)
+
+# M
+- Mike McDonald
+- Gunnar Morling - [@gunnarmorling](https://github.com/gunnarmorling/)
+- Rachel Myers - [@rachelmyers](https://github.com/rachelmyers)
```

Comment:
Not sure if this PR is the right place to add myself (I'm happy to open a separate PR if that's better)
```suggestion
- Rachel Myers - [@rachelmyers](https://github.com/rachelmyers)
- Calum Murray - [@Cali0707](https://github.com/Cali0707/)
```